### PR TITLE
[ISS-41] tests core factories

### PR DESCRIPTION
**Pull request by jlatusek**

### CODE REVIEW

**Comment by jlatusek** [comment]
dependency #75

**Comment by Hamilkar247** [comment]
add prefix [ISS-41] to commits .

**Review by Hamilkar247 — COMMENTED** [review]
if your branch is ready to review, please assigne me on the branch

**Comment by Hamilkar247** [comment]
@jlatusek

**Comment by Hamilkar247** [comment]
In the future, I think you should have only one branch=one issue, if branch will be to big , create new issue 'tests core factories', why? I think a few of this commit could be a duplicate with #75 but i am not sure without analyses and compare both. Probably it won't give errors but ...

File: bostardecksweb/test_settings.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from .settings import *
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
```
[the remark this answers is not in the document]
Comment (jlatusek) [reply bostardecksweb/test_settings.py:5]: Yes, it is faster

**Comment by jlatusek** [comment]
Yes there is duplication in commits because this pull request depend on #75

**Comment by jlatusek** [comment]
@Hamilkar247

File: bostardecksweb/test_settings.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hamilkar247) [reply bostardecksweb/test_settings.py:5]: ok

**Comment by Hamilkar247** [comment]
@jlatusek  - please do git rebase origin/master

**Pull request merged**